task-manager-backend/src/task/mongo/task-mongo.js

```diff
@@ -0,0 +1,69 @@
+const Task = require("./task-model");
+
+const {
+    TASK_CREATE_FAILED,
+    TASK_FETCH_FAILED,
+    TASK_UPDATE_FAILED,
+    TASK_DELETE_FAILED,
+} = require("../../commons/constants");
+
+const createTask = async (data) => {
+    try {
+        return await Task.create(data);
+    } catch (error) {
+        throw new Error(TASK_CREATE_FAILED);
+    }
+};
+
+const getAllTasks = async (userId) => {
```

Comment:
this actually gets task by user id so lets name it accordingly

Reply:
okay